packages/runtime/src/lib/txn.ts

```diff
@@ -280,6 +278,64 @@ export function isEncTxAssetConfig(txn: EncTx): boolean {
 	);
 }
 
+/**
+ * Check if given encoded transaction obj is asset creation
+ * @param txn Encoded EncTx Object
+ */
+export function isEncTxAssetCreate(txn: EncTx): boolean {
+	return (
+		txn.type === TransactionTypeEnum.ASSET_CONFIG && // type should be asset config
+		txn.caid === undefined && // assetIndex should be undefined
+		txn.apar !== undefined // assetParameters should not be undefined
+	);
+}
+/**
+ * Checks if given encoded transaction obj is asset reconfiguration
+ * @param txn Encoded EncTx Object
+ */
+export function isEncTxAssetReconfigure(txn: EncTx): boolean {
+	return (
+		txn.type === TransactionTypeEnum.ASSET_CONFIG && // type should be asset config
+		txn.caid !== undefined && // assetIndex should be undefined
+		txn.apar !== undefined && // assetParameters should not be undefined
+		(txn.apar.m !== undefined || // manager
+			txn.apar.f !== undefined || // freeze
+			txn.apar.c !== undefined || // clawback
+			txn.apar.r !== undefined) // reserve
+	);
+}
+/**
+ * Checks if given encoded transaction obj is asset revoke
+ * @param txn Encoded EncTx Object
+ */
+export function isEncTxAssetRevoke(txn: EncTx): boolean {
```

Comment:
How about create function like `getAlgoTransactionType(txn: EncTx): TransactionTypeEnum` This function will return algo-builder type of txn.

Reply:
Good idea

Reply:
@sczembor You forgot it. We can do it on another PR 🥰